### DISCUSSION TITLE
Missing quotes when generating default string array values

### DISCUSSION THREAD

**Discussion by edpri** [quarkiverse/quarkus-openapi-generator#905] [Q&A]
Hi, having this json spec:

```json
"realesrgan_enabled_models": {
            "title": "Realesrgan Enabled Models",
            "type": "array",
            "items": {},
            "description": "Select which Real-ESRGAN models to show in the web UI.",
            "default": [
              "R-ESRGAN 4x+",
              "R-ESRGAN 4x+ Anime6B"
            ]
},
```

I wonder if generated code is ok, i see that spec doesn't specify array item's types to be strings, but still it could be taken automatically as default values in spec give strong indication.

I mean, that's probably fine that generated field is of type Object, but when its trying to specify default values in java, it won't quote them and error happens:

```java
public Options addRealesrganEnabledModelsItem(Object realesrganEnabledModelsItem) {
        if (this.realesrganEnabledModels == null){
            realesrganEnabledModels = new ArrayList<>(Arrays.asList(R-ESRGAN 4x+, R-ESRGAN 4x+ Anime6B));
        }
        this.realesrganEnabledModels.add(realesrganEnabledModelsItem);
        return this;
}
```

Is that intended, should I fix spec or is it apigen bug?

Thanks

**Comment by ricardozanini** [comment]
It seems like a bug. It looks like the generator is considering this list as an Enum. Can you send a PR?
